### ISSUE TITLE
Gravestones not spawning, all my items just go everywhere like normal

### ISSUE THREAD

**Issue by Lucas1044** [OpenMods/OpenBlocks#1028]
I am playing on 1.7.10 and I put the OpenBlocks mod into my CrazyCraft 4 modpack, and all of the items in the mod show up and seem to be working, but when I die my items just go flying everywhere like normal minecraft, and no gravestone shows up. I have check the gamerule and gravestones are enabled, but it still does not work.

**Comment by boq** [comment]
Enable ` /om_config_s set openblocks debug gravesdebug true`, try to spawn grave and post log file (there should be section starting with `openblocks.common.PlayerInventoryStore.onPlayerDeath`

**Comment by Lucas1044** [comment]
> Enable ` /om_config_s set openblocks debug gravesdebug true`, try to spawn grave and post log file (there should be section starting with `openblocks.common.PlayerInventoryStore.onPlayerDeath`

I did the command and nothing seems to have changed. These are the logs for when I gave myself dirt to see if it would go into the grave when I died and then I did/kill. It didnt go into a grave and no grave spawned.

[08:06:04] [Server thread/INFO]: [ItsDropster: Given [Dirt] * 64 to ItsDropster]
[08:06:04] [Client thread/INFO]: [CHAT] Given [Dirt] * 64 to ItsDropster
[08:06:06] [Client thread/INFO]: net.minecraft.client.gui.inventory.GuiInventory
[08:06:06] [Client thread/INFO]: net.minecraft.inventory.ContainerPlayer
[08:06:07] [Client thread/INFO]: net.minecraft.client.gui.GuiChat
[08:06:08] [Server thread/INFO]: openblocks.common.PlayerInventoryStore.onPlayerDeath(PlayerInventoryStore.java:312): Storing post-mortem inventory into C:\Users\Gwar  Gumm\AppData\Roaming\.crazycraft4\saves\New World--\data\inventory-ItsDropster-2020-12-15_08.06.08-death-0.dat. It can be restored with command '/ob_inventory restore ItsDropster ItsDropster-2020-12-15_08.06.08-death-0'
[08:06:08] [Server thread/INFO]: ItsDropster fell out of the world
[08:06:08] [Client thread/INFO]: [CHAT] ItsDropster fell out of the world
[08:06:08] [Client thread/INFO]: [CHAT] Ouch! That looked like it hurt
[08:06:08] [Client thread/INFO]: net.minecraft.client.gui.GuiGameOver
[08:06:09] [Client thread/INFO]: ItsDropster died at x:172, y:64, z:243. Deathpoint created: true

**Comment by boq** [comment]
If graves are enabled, there should be more lines with `PlayerDeathHandler` in them after that block

**Comment by Lucas1044** [comment]
there is nothing else after that, is there something to do to fix that?

**Comment by boq** [comment]
Hmm, those look like logs from console, can you get one from file?

**Comment by Lucas1044** [comment]
I'm not sure what you mean, the way I am getting these is by going into my roaming, opening .crazycraft4, opening logs, and clikcing the text file called latest.

**Comment by boq** [comment]
Oh, this is already from `client-latest`. If it was vanilla launcher, it would include lines starting including `[Server thread/DEBUG]`, but I only see `INFO`, so I assumed it's console, which does not print that. If it isn't there, then some other mod it completely removing parts on Forge required for grave drop, since otherwise that code is guaranteed to log something.

For more detailed procedure, see https://openmods.info/faq.html#ob-0

**Comment by Lucas1044** [comment]
what other file would i use to see the logs other than the latest file

**Comment by boq** [comment]
It is supposed to be in `client-latest.log`. If it isn't there, I suspect some setting from launcher that filters it. If you are using vanilla launcher, then I'm a bit puzzled.

**Comment by Lucas1044** [comment]
i am not I am using voids wrath launcher

**Comment by boq** [comment]
Sorry, I can't help you with third-party component like that. If those logs were not filtered, there would be line starting with something like `[Server thread/DEBUG] [OpenMods/]: openblocks.common.PlayerDeathHandler` saying exactly why grave didn't spawn.

**Comment by Lucas1044** [comment]
is there any way to launch this modpack without using the voids wrath launcher

**Comment by RConTRS** [comment]
yes u can, simply set ur instance folder path to CC4 pack folder, navigate to the pack folder or via the void launcher and copy the path then paste it into the instance folder/path in the launcher u use.

i really want to fix this issue of graves not spawning and all my items drops just like if there are no graves.
